src/ergw_aaa_radius.erl

```diff
@@ -359,8 +363,12 @@ vendor_ituma('WLAN-Station-RSSI ', #{'WLAN-Station-RSSI' := Value}, Attrs) ->
     [{?'IM_UE_Sta_RSSI', Value}|Attrs];
 vendor_ituma('WLAN-AP-RSSI', #{'WLAN-AP-RSSI' := Value}, Attrs) ->
     [{?'IM_UE_Wwan_RSSI', Value}|Attrs];
+vendor_ituma('LI-Location', #{'IM_LI_Location' := LocationStr}, Attrs) ->
+    ?LOG(debug, "Adding LI-Location (decimal string): ~s", [LocationStr]),
+    [{?'IM_LI_Location', LocationStr}|Attrs];
```

Comment:
I do not see a valid reason to add this, the existing location handling is fully able to handle this.

Reply:
The location provider will right now provide an opaque string. Sure we can parse it, ad put it back into the format required here, but it's completely unnecessary.